packages/react-dom/src/events/plugins/ModernSelectEventPlugin.js

```diff
@@ -144,6 +146,29 @@ function constructSelectEvent(nativeEvent, nativeEventTarget) {
   return null;
 }
 
+function isListeningToEvents(
+  events: Array<string>,
+  mountAt: Document | Element,
+): boolean {
+  const listenerMap = getEventListenerMap(mountAt);
+  for (let i = 0; i < events.length; i++) {
+    const event = events[i];
+    if (!isListeningToEvent(event, mountAt, listenerMap)) {
+      return false;
+    }
+  }
+  return true;
+}
+
+function isListeningToEvent(
+  registrationName: string,
+  mountAt: Document | Element,
+  listenerMap?: ElementListenerMap,
```

Comment:
This optional param seems awkward. I think it would be better just to inline the `listenerMap.has(registrationName)` check in `isListeningToEvents` and leave off this new param for the other callers.